### PR TITLE
[Doc]update run visualize screenshot

### PR DESCRIPTION
**Pull request by D-W-**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

image

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: docs/how-to-guides/run-and-evaluate-a-flow/index.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ pf run visualize -n "my_first_run,my_first_eval_run"
 
 A web browser will be opened to show the visualization result.
 
-![q_0](../../media/how-to-guides/visualize_run.png)
+![q_0](../../media/how-to-guides/run_visualize.png)
```
Comment (crazygao) [inline docs/how-to-guides/run-and-evaluate-a-flow/index.md:158]: delete the old png ?

Comment (wangchao1230) [reply docs/how-to-guides/run-and-evaluate-a-flow/index.md:158]: also change the place reference it

**Review by brynn-code — APPROVED** [review]

**Review by crazygao — DISMISSED** [review]

File: docs/how-to-guides/run-and-evaluate-a-flow/index.md (hunk 1 above)
Comment (crazygao) [reply docs/how-to-guides/run-and-evaluate-a-flow/index.md:158]: There are two visualized pictures, do you need to change another one?

Comment (D-W-) [reply docs/how-to-guides/run-and-evaluate-a-flow/index.md:158]: Good catch, update.

**Review by brynn-code — APPROVED** [review]

**Pull request merged**